components/header/header.tsx

```diff
@@ -27,6 +26,12 @@ const DevelopmentDropdown = dynamic(
     ssr: false,
   }
 );
+const DesignDropdown = dynamic(
+  () => import("../design-dropdown/design-dropdown"),
+  {
+    ssr: false,
```

Comment:
I am just curious about which component is causing the issue. Is it the Popper again? It will be good to know to give feedback to PF.

Reply:
@hyperkid123 That's @karelhala 's work